src/screens/Profile/index.tsx

```diff
@@ -0,0 +1,212 @@
+import {launchImageLibrary} from 'react-native-image-picker';
+import {isEqual} from 'lodash';
+import commonStyles from '../../commonStyles';
+import Button from '../../components/Button/Button';
+import {ButtonColor} from '../../components/Button/Button.types';
+import TextInput from '../../components/TextInput';
+import ScreenWithKeyboard from '../../components/ScreenWithKeyboard';
+import {MODAL_ROUTES} from '../../constants/routes';
+import {
+  PROFILE_IMAGE,
+  PROFILE_IMAGE_HEIGHT,
+  PROFILE_IMAGE_WIDTH,
+  WHITE,
+} from '../../constants';
+import {retreiveToken} from '../../helpers/access';
+import styles from './styles';
+
+import mockData from './data.json';
+import ProfileIcon from '../../assets/avatars/Profile.svg';
+
+type Profile = {
+  name?: string;
+  phone?: string;
+  city?: string;
+  street?: string;
+  flat?: string;
+};
+
+type Error = {
+  name?: string;
+};
+
+const Profile: FC = () => {
+  const [initialProfileData, setInitialProfileData] = useState<Profile>({});
+  const [profileData, setProfileData] = useState<Profile>({});
+  const [image, setImage] = useState<string | null | undefined>(null);
+  const [errors, setErrors] = useState<Error>({});
+  const navigation = useNavigation();
+
+  useEffect(() => {
+    const bootstrapAsync = async () => {
+      try {
+        const profileImage = await AsyncStorage.getItem(PROFILE_IMAGE);
+        setImage(profileImage);
+
+        const accessToken = await retreiveToken();
+
+        if (accessToken) {
+          // ToDo: use Account API and uncomment it
+          /* const options = {
+            method: 'GET',
+            headers: {
+              'Content-Type': 'application/json',
+              Authorization: `Bearer ${accessToken}`,
+            },
+          };
+          const response = await fetch(`${API_URL}/account`, options);
+          const parsedResponse = await response.json();
+          setProfileData(parsedResponse.data);
+          */
+
+          const profile = {
+            name: mockData.attributes.firstname,
+            phone: mockData.attributes.phone,
+            city: mockData.attributes.city,
+            street: mockData.attributes.address1,
+            flat: mockData.attributes.address2,
+          };
+          setProfileData(profile);
+          setInitialProfileData(profile);
+        }
+      } catch (error) {
+        console.error(error);
+      }
+    };
+
+    bootstrapAsync();
+  }, []);
+
+  const onChangeName = (name: string) => {
+    setProfileData({...profileData, name});
+  };
+
+  const onChangePhone = (phone: string) => {
+    setProfileData({...profileData, phone});
+  };
+
+  const onChangeCity = (city: string) => {
+    setProfileData({...profileData, city});
+  };
+
+  const onChangeStreet = (street: string) => {
+    setProfileData({...profileData, street});
+  };
+
+  const onChangeFlat = (flat: string) => {
+    setProfileData({...profileData, flat});
+  };
```

Comment:
i suppose that it can be replaced with one function:
```js
const onChangeProfileData = (value: string, name: string) =>  {
  setProfileData(prevData => ({...prevData, [name]: value}));
};
```